app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1><%= @recipe.title %></h1>
+<h1><%= @recipe.prep_time %></h1>
+<h1><%= @recipe.cooking_time %></h1>
+<h1><%= @recipe.total_time %></h1>
+<h1><%= @recipe.difficulty %></h1>
+<h1><%= @recipe.utensil %></h1>
+<h1><%= @recipe.instruction %></h1>
+
+
+<%# Still just the raw basics, have to add dietary requirements, ingredients, frontend, buttons to redirect%>
```

Comment:
H1 is for page header only :)

Reply:
like i said, it is just the basic to show something while i was working, i know everything should not be h1, i just copy and pasted, cause i am sneaky

Reply:
(so i will not change that)

Reply:
Ah ok sorry missed it after 5 pages :) Maybe put it in comment on the code? so we remmeber? just suggestion for next time :)